src/components/Hosts/UpdateStatusModal.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from "react";
+import { Select, Input } from "antd";
+import styled from "@emotion/styled";
+import { useMutation } from "@apollo/react-hooks";
+import { Modal } from "components/Modal";
+import { Button } from "components/Button";
+import {
+  UpdateHostStatusMutation,
+  UpdateHostStatusMutationVariables,
+} from "gql/generated/types";
+import { UPDATE_HOST_STATUS } from "gql/mutations";
+import { useBannerDispatchContext } from "context/banners";
+import { UpdateHostStatus } from "types/host";
+import { Body } from "@leafygreen-ui/typography";
+
+const { Option } = Select;
+const { TextArea } = Input;
+
+interface Props {
+  visible: boolean;
+  dataCy: string;
+  hostIds: string[];
+  closeModal: () => void;
+}
+
+export const UpdateStatusModal: React.FC<Props> = ({
+  visible,
+  dataCy,
+  hostIds,
+  closeModal,
+}) => {
+  const dispatchBanner = useBannerDispatchContext();
+
+  const [status, setHostStatus] = useState<UpdateHostStatus | null>(null);
```

Comment:
You can remove the `| null` from the type of the useState() I believe the typescript compiler will ignore it any ways simply using `useState<UpdateHostStatus>(null)` should accomplish the same effect.
![image](https://user-images.githubusercontent.com/4605522/89921699-057b4f00-dbcc-11ea-8c9e-fa9003ebfe61.png)
![image](https://user-images.githubusercontent.com/4605522/89921755-162bc500-dbcc-11ea-9e6d-f4085714b741.png)

Reply:
ok